fmriprep/cli/run.py

```diff
@@ -5,7 +5,6 @@
 fMRI preprocessing workflow
 =====
 """
-from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Is the plan to completely discontinue Py2 support? At least a couple months ago, we were accepting patches from users who needed Py2, as long as they didn't complicate the code.